specs/http-client/index.html

```diff
@@ -233,23 +238,23 @@ <h2>The <dfn>http:send-request</dfn> function</h2>
       <ul>
         <li><code>$request</code> contains the various parameters of the request, for instance the
           HTTP method to use or the HTTP headers. Among other things, it can also contain the other
-          param's values: the URI and the bodies. If they are not set as parameter to the function,
-          their value in $request, if any, is used instead. See the following section for the
-          detailed definition of the http:request element. If the parameter does not follow the
-          grammar defined in this spec, this is an error <a href="#dfn-errHC005">err:HC005</a>.</li>
-        <li><code>$href</code> is the HTTP or HTTPS URI to send the request to. It is an xs:anyURI,
+          parameter’s values: the URI and the body(s). If they are not set as a parameter to the function,
+          their value in <code>$request</code>, if any, is used instead. See the <a href="#request-element" class="sectionRef"></a> for the
+          definition of the <a>http:request</a> element. If the parameter does not follow the
+          grammar defined in this spec, the error <a>err:HC005</a> MUST be raised.</li>
+        <li><code>$href</code> is the HTTP or HTTPS URI to send the request to. It is an <code>xs:anyURI</code>,
           but is declared as a string to be able to pass literal strings (without requiring to
-          explicitly cast it to an xs:anyURI).</li>
+          explicitly cast it to an <code>xs:anyURI</code>).</li>
```

Comment:
"to be able to pass literal strings... xs:anyURI" > "so that literal strings may be used; in other words, the parameter does not need to be explicitly cast as xs:anyURI."